projects/zxing-scanner/src/lib/zxing-scanner.component.ts

```diff
@@ -535,21 +536,18 @@ export class ZXingScannerComponent implements AfterViewInit, OnDestroy {
   }
 
   /**
-   * Starts the scanner with the first available device.
+   * Starts the scanner with the back camera otherwise take the first
```

Comment:
```suggestion
   * Starts the scanner with the back camera otherwise take the last
```

Reply:
It's indeed in fact the last, I didn't really check what was written before with what happens.

Reply:
Old comment, I forgot to change it probably because of the same reason you did, hehe.